R/stochastic_process.R

```diff
@@ -486,7 +454,20 @@ stochastic_process_validate <- function(con, touchpoint, scenarios, in_path,
   if (!file.exists(in_path)) {
     stop(sprintf("Input path not found: %s", in_path))
   }
-  stopifnot(file.exists(in_path))
+
+  assert_scalar_character(out_path)
```

Comment:
why has this function picked up a dependency on `out_path` in addition to `pre_aggregation_path`? This is just a general tidyup?